microstrain_inertial_msgs/CMakeLists.txt

```diff
@@ -25,11 +25,7 @@ if(NOT EXISTS "${COMMON_DIR}/msg")
   # Initialize and update the submodule
   execute_process(
     WORKING_DIRECTORY "${${PROJECT_NAME}_SOURCE_DIR}"
-    COMMAND ${CMAKE_COMMAND} -E env ${GIT_EXECUTABLE} submodule init
-  )
-  execute_process(
-    WORKING_DIRECTORY "${${PROJECT_NAME}_SOURCE_DIR}"
-    COMMAND ${CMAKE_COMMAND} -E env ${GIT_EXECUTABLE} submodule update --recursive
+    COMMAND ${CMAKE_COMMAND} -E env ${GIT_EXECUTABLE} submodule update --init --recursive
```

Comment:
Looks like you need to merge the `ros2` branch into your branch as this change was already made